scripts/common/common.sh

```diff
@@ -424,7 +424,18 @@ function report_install_docker() {
 }
 
 function report_install_containerd() {
-    addon_install "containerd" "$CONTAINERD_VERSION"
+    if [ -z "$CURRENT_KUBERNETES_VERSION" ] || [ ! -f "/usr/bin/containerd" ]; then
```

Comment:
```suggestion
    if [ -z "$CURRENT_KUBERNETES_VERSION" ] || ! commandExists containerd; then
```
There's a built in command to check if command exist: `commandExists`.